### ISSUE TITLE
Nexus data directory already in use: /nexus-data

### ISSUE THREAD

**Issue by alexandrst88** [travelaudience/kubernetes-nexus#43]
When upgrade chart via helm, the new pod is stuck, with RollingUpdate strategy due to 
```
Nexus data directory already in use: /nexus-data
```

**Comment by pires** [comment]
@jeff-knurek @TAvardit as the Helm experts in the house, can you please help?

**Comment by TAvardit** [comment]
@alexandrst88 - The problem is caused by having a rollingUpdate strategy with default settings (MaxSurge 25% maxUnavailable 25%) , one replica and PV. An ability to change these settings was added to chart version `1.10.0` (https://github.com/helm/charts/tree/master/stable/sonatype-nexus), so you can either upgrade your chart and edit the values or update you live deployment manifest to be set to : 
```
maxSurge: 0
maxUnavailable: 1
```

**Comment by mwy-bi-web** [comment]
The chart has no option to set MaxSurge/maxUnavailable. So this is still an issue.

**Comment by TAvardit** [comment]
Actually, if no strategy was specified the default strategy is `rollingUpdate`.
Did you experience issue deploying the chart without changing the empty object of `deploymentStrategy` ?

**Comment by mwy-bi-web** [comment]
Yes. The default is maxSurge: 25%,  maxUnavailable: 25%. So with 1 replicas, nexus is first started and then stopped leading to this issue.

**Comment by TAvardit** [comment]
Yes, I didn't want to change the default `deploymentStrategy` but to allow users to change it based on their need.
